apps/emqx_resource/src/schema/emqx_resource_schema.erl

```diff
@@ -142,6 +160,7 @@ request_ttl(_) -> undefined.
 enable_batch(type) -> boolean();
 enable_batch(required) -> false;
 enable_batch(default) -> true;
+enable_batch(importance) -> ?IMPORTANCE_HIDDEN;
```

Comment:
This one is already deprecated, so maybe there's no need to hide?

Reply:
deprecated and hidden do  not conflict each other.